deployment/ansible/templates/epoch.yaml.j2

```diff
@@ -2,15 +2,18 @@
 sync:
     port: {{ config.sync.port }}
 
+{% if configure_peers|default(false) %}
```

Comment:
I guess we have to set the default to `true` because all environments except UAT needs it.

Sorry for mislead you.